backend/src/util/virtualTransactions.ts

```diff
@@ -68,12 +68,12 @@ const virtualDecayTransaction = (
     userId: -1,
     previous: -1,
     typeId: TransactionTypeId.DECAY,
-    amount: decay.decay ? decay.decay.toDecimalPlaces(2, Decimal.ROUND_FLOOR) : new Decimal(0), // new Decimal(0), // this kinda is wrong, but helps with the frontend query
+    amount: decay.decay ? decay.roundedDecay : new Decimal(0), // this kinda is wrong, but helps with the frontend query
```

Comment:
can we move this comment to the actual calculation in `util/decay.ts` explaingin why and how we do that, please?

Reply:
I do not know what this comment is for. This is exactly the point why I do not like comments in the code. They pollute the code and create confusion.